src/ICSharpCode.SharpZipLib/Tar/TarEntry.cs

```diff
@@ -379,10 +379,14 @@ public void GetFileTarHeader(TarHeader header, string file)
 			string name = file;
 
 			// 23-Jan-2004 GnuTar allows device names in path where the name is not local to the current directory
+			// 04-Apr-2019 Removed this block, as it creates different behavior based the location of the executing
+			//             assembly.
+			/*
```

Comment:
The purpose of this block is to do exactly that. While I do agree that this may not belong here, just removing it will just break code relying on this behaviour.